<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelemetryFrame</class>
 <widget class="QWidget" name="TelemetryFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>309</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Telemetry Chart</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="toolBarLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="ClickableLabel" name="lbCurrent">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbTotal">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="eNotes">
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Notes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
  <action name="aLast">
   <property name="icon">
    <iconset resource="../../resources/standard-icons.qrc">
     <normaloff>:/icons/old/transport_range.png</normaloff>:/icons/old/transport_range.png</iconset>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
   <property name="toolTip">
    <string>Load the most recent flight</string>
   </property>
  </action>
  <action name="aDelete">
   <property name="icon">
    <iconset resource="../../resources/standard-icons.qrc">
     <normaloff>:/icons/old/edit-delete.png</normaloff>:/icons/old/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete current flight</string>
   </property>
  </action>
  <action name="aEdit">
   <property name="icon">
    <iconset resource="../../resources/standard-icons.qrc">
     <normaloff>:/icons/old/text-plain.png</normaloff>:/icons/old/text-plain.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Open in external editor</string>
   </property>
  </action>
  <action name="aExport">
   <property name="icon">
    <iconset resource="../../resources/standard-icons.qrc">
     <normaloff>:/icons/old/ark_extract.png</normaloff>:/icons/old/ark_extract.png</iconset>
   </property>
   <property name="text">
    <string>Export data..</string>
   </property>
   <property name="toolTip">
    <string>Export data..</string>
   </property>
  </action>
  <action name="aReport">
   <property name="icon">
    <iconset resource="../../resources/standard-icons.qrc">
     <normaloff>:/icons/old/view-certificate-export.png</normaloff>:/icons/old/view-certificate-export.png</iconset>
   </property>
   <property name="text">
    <string>Report</string>
   </property>
   <property name="toolTip">
    <string>Create Report</string>
   </property>
  </action>
  <action name="aCut">
   <property name="icon">
    <iconset resource="../../resources/standard-icons.qrc">
     <normaloff>:/icons/old/edit-cut.png</normaloff>:/icons/old/edit-cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut file to fit current view</string>
   </property>
  </action>
  <action name="aFilter">
   <property name="icon">
    <iconset resource="../../resources/standard-icons.qrc">
     <normaloff>:/icons/old/document-sign.png</normaloff>:/icons/old/document-sign.png</iconset>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
   <property name="toolTip">
    <string>Show filter configuration...</string>
   </property>
  </action>
  <action name="aFullScreen">
   <property name="icon">
    <iconset resource="../../resources/standard-icons.qrc">
     <normaloff>:/icons/old/view-fullscreen.png</normaloff>:/icons/old/view-fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="toolTip">
    <string>Toggle Full Screen</string>
   </property>
  </action>
  <action name="aPrev">
   <property name="icon">
    <iconset resource="../../resources/standard-icons.qrc">
     <normaloff>:/icons/old/nudge_left.png</normaloff>:/icons/old/nudge_left.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="toolTip">
    <string>Load previous flight</string>
   </property>
  </action>
  <action name="aNext">
   <property name="icon">
    <iconset resource="../../resources/standard-icons.qrc">
     <normaloff>:/icons/old/nudge_right.png</normaloff>:/icons/old/nudge_right.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Load next flight</string>
   </property>
  </action>
  <action name="aFiles">
   <property name="icon">
    <iconset resource="../../resources/standard-icons.qrc">
     <normaloff>:/icons/old/document-preview-archive.png</normaloff>:/icons/old/document-preview-archive.png</iconset>
   </property>
   <property name="text">
    <string>Show Files</string>
   </property>
   <property name="toolTip">
    <string>Show file manager...</string>
   </property>
  </action>
  <action name="aReload">
   <property name="icon">
    <iconset resource="../../resources/standard-icons.qrc">
     <normaloff>:/icons/old/transport_loop.png</normaloff>:/icons/old/transport_loop.png</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload current file</string>
   </property>
  </action>
  <action name="aSplitV">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/standard-icons.qrc">
     <normaloff>:/icons/old/view-split-top-bottom.png</normaloff>:/icons/old/view-split-top-bottom.png</iconset>
   </property>
   <property name="text">
    <string>Split Vertically</string>
   </property>
   <property name="toolTip">
    <string>Split view vertically</string>
   </property>
  </action>
  <action name="aPlay">
   <property name="icon">
    <iconset resource="../../resources/standard-icons.qrc">
     <normaloff>:/icons/old/transport_play.png</normaloff>:/icons/old/transport_play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play telemetry data...</string>
   </property>
  </action>
  <action name="aFilterUAV">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/standard-icons.qrc">
     <normaloff>:/icons/old/view-filter.png</normaloff>:/icons/old/view-filter.png</iconset>
   </property>
   <property name="text">
    <string>Filter UAV</string>
   </property>
   <property name="toolTip">
    <string>Filter data files by UAV name...</string>
   </property>
  </action>
  <action name="avSTD">
   <property name="text">
    <string>STD</string>
   </property>
   <property name="toolTip">
    <string>Show common plots</string>
   </property>
  </action>
  <action name="avIMU">
   <property name="text">
    <string>IMU</string>
   </property>
   <property name="toolTip">
    <string>Show IMU sensors</string>
   </property>
  </action>
  <action name="avCTR">
   <property name="text">
    <string>CTR</string>
   </property>
   <property name="toolTip">
    <string>Show controls</string>
   </property>
  </action>
  <action name="avCLR">
   <property name="text">
    <string>CLR</string>
   </property>
   <property name="toolTip">
    <string>Clear visible plots</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">ClickableLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/standard-icons.qrc"/>
 </resources>
 <connections/>
</ui>
